waltz-tools/src/main/java/com/wepay/waltz/tools/zk/ZooKeeperCli.java

```diff
@@ -535,6 +535,11 @@ protected void processCmd(CommandLine cmd) throws SubCommandFailedException {
                 int adminPort = Integer.parseInt(cmd.getOptionValue("storage-admin-port"));
                 int groupId = Integer.parseInt(cmd.getOptionValue("group"));
 
+                String[] storageHostAndPort = storage.split(":");
+                if (storageHostAndPort.length != 2) {
+                    throw new IllegalArgumentException("Storage must be in format of host:admin_port");
```

Comment:
I think it should be in host:port, we have -a for admin port.

Reply:
> I think it should be in host:port, we have -a for admin port.

Ohh yes, I'll modify that